File: exercicios/Python/2235/2235.py
v = input().split()

a = int(v[0])
b = int(v[1])
c = int(v[2])

if((a-b == 0) or (a-c == 0) or (b-c == 0)):
    print('S')
else:
    if((a+b-c == 0) or (b-a+c == 0) or (c-a+b == 0)):
        print('S')
    elif((a-b-c == 0) or (b-a-c == 0) or (c-a-b == 0)):
        print('S')
    else:
        print('N')

File: exercicios/JavaScript/2235/2235.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var [a,b,c] = lines.shift().split(' ').map(Number);

a = parseInt(a)
b = parseInt(b)
c = parseInt(c)

if((a-b == 0) || (a-c == 0) || (b-c == 0)){
    console.log('S')}

else{
    if((a+b-c == 0) || (b-a+c == 0) || (c-a+b == 0)){
        console.log('S')}
    else if((a-b-c == 0) || (b-a-c == 0) || (c-a-b == 0)){
        console.log('S')
    }else{
        console.log('N')
    }
}
